### ISSUE TITLE
Override and register

### ISSUE THREAD

**Issue by MatsG23** [JorelAli/CommandAPI#232]
A new method of `CommandAPICommand` which enables you to override a command directly instead of using
```kotlin
CommandAPI.unregister("avanillacommand", true) 
CommandAPICommand("avanillacommand")
     .executes(CommandExecutor { ... })
     .register()
```

_Example_
```kotlin
CommandAPICommand("avanillacommand")
     .executes(CommandExecutor { ... })
     .override()
```

**Comment by JorelAli** [comment]
Not a bad idea. The whole "unregistering a command" code needs more work (sometimes doesn't work), so I'll use this issue to keep track of that as well.

**Comment by JorelAli** [comment]
Implemented for 6.3.1 release.

**Comment by JorelAli** [comment]
Implemented in release 6.3.1
